coops-ui/src/interfaces/address-interfaces.ts

```diff
@@ -1,29 +1,29 @@
 import Vue from 'vue'
 
 export interface BaseAddressType extends Vue {
-  $refs: any
+  $refs: any;
```

Comment:
This is interesting! Are you able to elaborate on why the semi-colons in this file?

Reply:
This is the normal way that properties (indeed, any JS statement) is terminated. However, because this is actually TS, there is some flexibility in the syntax. But I prefer terminating my statements explicitly to avoid any surprises!

Reply:
Fair enough and i couldn't agree more. Personally i would love to see the semi colons everywhere :)  I guess the question should have been more along the lines " Why here in the interfaces but not elsewhere? "

Reply:
Well, for one, one of the interfaces used semi-colons and the others didn't, so I made it consistent. But you're right, a lot (most?) of our other code doesn't use semi-colons for properties. So this could go either way. This particular change seemed like a good idea at the time, hah.

Reply:
Works for me, i was just curious!